app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class UserMailer < ApplicationMailer
+  default from: "from@example.com"
+
+  def welcome_email(user)
+    @user = user
+    mail(to: @user.email, subject: 'Sample Email')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.